### DISCUSSION TITLE
plotly.write_image(fig) keeps executing without end

### DISCUSSION THREAD

**Discussion by AnupamSaraph** [microsoft/vscode-jupyter#10108] [Questions and Answers]
While the code below would earlier produce png, svg, or html based on the extension, it is going into an endless execution of the cell and producing no output even after three hours.

```
import plotly.graph_objects as go
import numpy as np
from pathlib import Path

file_name = Path.cwd().joinpath("images")
if not f.is_dir(): f.mkdir()
file_name = file_name.joinpath("fig1.png")

fig = go.Figure(go.Scatter(x=np.linspace(1,10,100), y=np.sin(np.linspace(-np.pi,np.pi, 100))))
fig.show()
fig.write_image(file_name, engine='auto')

```
I am using plotly 5.8.0 and kaleido 0.2.1

https://user-images.githubusercontent.com/1920565/169955341-498b2543-0702-4dd1-a531-09ab0af4dcc8.mp4

**Comment by rchiodo** [comment]
This should be a problem with the plotly package. It's never finishing. 

It did work for me though. I have version 5.8.0 too. Maybe some other package is out of date? I had to install `kaleido` (version 0.2.1) to get this to work. 

![image](https://user-images.githubusercontent.com/19672699/170358941-2e73af32-82b7-4d32-8d70-ff3d7071c445.png)

**Reply by DonJayamanne** [reply]
I can confirm it works for me too
